nautobot/docs/user-guide/platform-functionality/jobs/index.md

```diff
@@ -2,81 +2,35 @@
 
 Jobs are a way for users to execute custom logic on demand from within the Nautobot UI. Jobs can interact directly with Nautobot data to accomplish various data creation, modification, and validation tasks, such as:
 
-* Automatically populate new devices and cables in preparation for a new location deployment
-* Create a range of new reserved prefixes or IP addresses
-* Fetch data from an external source and import it to Nautobot
-* Check and report whether all top-of-rack switches have a console connection
-* Check and report whether every router has a loopback interface with an assigned IP address
-* Check and report whether all IP addresses have a parent prefix
+- Automatically populate new devices and cables in preparation for a new location deployment
+- Create a range of new reserved prefixes or IP addresses
+- Fetch data from an external source and import it to Nautobot
+- Check and report whether all top-of-rack switches have a console connection
+- Check and report whether every router has a loopback interface with an assigned IP address
+- Check and report whether all IP addresses have a parent prefix
 
 ...and so on. Jobs are Python code and exist outside of the official Nautobot code base, so they can be updated and changed without interfering with the core Nautobot installation. And because they're completely customizable, there's practically no limit to what a job can accomplish.
 
+The following pages provide detailed guides for managing and running Jobs:
+
+- **[Managing Jobs](./managing-jobs.md)**: Enable/disable Jobs, override metadata, delete Jobs.
+- **[Job Scheduling & Approvals](./job-scheduling-and-approvals.md)**: Schedule Jobs to run in the future or regularly.
+- **[Job Queues](./jobqueue.md)**: Configure dedicated Celery or Kubernetes queues.
+- **[Job Buttons](./jobbutton.md)**: Run Jobs from detail views of Nautobot objects.
+- **[Job Hooks](./jobhook.md)**: Automatically trigger Jobs based on object changes.
+- **[Kubernetes Job Support](./kubernetes-job-support.md)**: Run Jobs within a Kubernetes cluster.
+
```

Comment:
A link to the developer side fits in here I think, just to aid navigation/discoverability. Feel free to edit the text below.

```suggestion
- **[Kubernetes Job Support](./kubernetes-job-support.md)**: Run Jobs within a Kubernetes cluster.
- **[Developing Jobs](../development/jobs)**: To write your own jobs or migrate them from previous Nautobot releases, check out the developer section.
```